tests/DCPS/XTypes/desktop.ini

```diff
@@ -0,0 +1,3 @@
+[LocalizedFileNames]
+RtpsTypeLookupService.cpp=@RtpsTypeLookupService.cpp,0
+RtpsTypeLookupService.h=@RtpsTypeLookupService.h,0
```

Comment:
You should remove this.

Reply:
This file is in this project is probably by mistake, due to some git manipulations. Can it be removed?